tests/runtime/filter_parser.c

```diff
@@ -709,10 +789,11 @@ TEST_LIST = {
     {"filter_parser_extract_fields", flb_test_filter_parser_extract_fields },
     {"filter_parser_reserve_data_off", flb_test_filter_parser_reserve_data_off },
     {"filter_parser_handle_time_key", flb_test_filter_parser_handle_time_key },
+    {"filter_parser_handle_time_key_with_time_zone", flb_test_filter_parser_handle_time_key_with_time_zone },
     {"filter_parser_ignore_malformed_time", flb_test_filter_parser_ignore_malformed_time },
     {"filter_parser_preserve_original_field", flb_test_filter_parser_preserve_original_field },
     {"filter_parser_first_matched_when_multiple_parser", flb_test_filter_parser_first_matched_when_mutilple_parser },
-    {"filter_parser_skip_empty_values_false", flb_test_filter_parser_skip_empty_values_false},
+    {"filter_parser_skip_empty_values_false", flb_test_filter_parser_skip_empty_values_false },
```

Comment:
Unexpected adding trailing space here?

Reply:
I was intending to make that one consistent with the other since they all have spaces, but I can remove this to make the diff simpler.